cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -6,32 +6,48 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [id="odo-notable-improvements_{context}"]
-== Notable improvements in `{odo-title}`{nbsp}{odo-ver}
+== Notable changes and improvements in `{odo-title}`{nbsp}{odo-ver}
+
+* A `--now` flag is added for `odo create`, `odo url` and `odo config`. 
++
+Run `odo url create --now` to create the URL in the configuration and propagate all the changes to the cluster
+* An `odo debug info` command is added. The command displays the information whether the debug mode is enabled for the component, whether the port-forward process is running, and information about ports used.
```

Comment:
The command displays shows if the debug mode is enabled for a component, if the port-forward process is running, and information about ports used.
Question: In 4.3, the ODO docs say that _Components created with odo run in the debug mode by default. _ Can the user turn off debug mode?

Reply:
@mburke5678 yes, the users can do that. I will check the assembly about debugging and if it's not there and update it with the instruction how to do that.